src/components/Clock/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Clock = () => {
+  const [currentTime, setCurrentTime] = React.useState(new Date());
+
+  React.useEffect(() => {
+    setInterval(moveClock, 1000);
+  }, []);
+
+  const moveClock = () => {
```

Comment:
Good you separated this method out from setInterval, because it makes it more testable.